apps/studio/src/server/modules/folder/__tests__/folder.router.test.ts

```diff
@@ -59,6 +60,9 @@ describe("folder.router", async () => {
       await expect(result).rejects.toThrowError(
         new TRPCError({ code: "UNAUTHORIZED" }),
       )
+      await expect(
+        db.selectFrom("AuditLog").selectAll().execute(),
+      ).resolves.toEqual([])
```

Comment:
super nit - checking count is 0 might be more readable than checking for empty array

Reply:
Updated in 9fff24c507dd19fca1f26ccd2fba956c074169d8.